Makefile

```diff
@@ -20,6 +20,23 @@ include $(top_srcdir)tools/build/Makefile.vars
 
 include $(top_srcdir)tools/build/Makefile.common
 
+ifneq ($(NODE_BINDINGS),)
+
+bins-out += node_bindings
```

Comment:
not sure `bins-out` is the best variable to append to.

@dorileo could you check?